api/handlers.go

```diff
@@ -55,7 +59,79 @@ func (s *Server) Pulse(ctx echo.Context, pulseNumber server.PulseNumberPathParam
 }
 
 func (s *Server) JetDropsByPulseNumber(ctx echo.Context, pulseNumber server.PulseNumberPathParam, params server.JetDropsByPulseNumberParams) error {
-	panic("implement me")
+	if !pulse.IsValidAsPulseNumber(int(pulseNumber)) {
+		response := server.CodeValidationError{
+			Code:               NullableString("400"),
```

Comment:
Maybe use `http.StatusText(http.StatusBadRequest)` instead of just string value?

Reply:
sure